saphir/src/utils.rs

```diff
@@ -558,6 +558,10 @@ where
     serde_urlencoded::from_str::<T>(query_str)
 }
 
+// #[cfg(feature = "tracing-instrument")]
```

Comment:
commented code?

Reply:
Oops, I cleaned up my test stuff afterward but left some behind it seems.
Removing this shortly